test/EntityManagerFactoryTest.php

```diff
@@ -26,15 +26,19 @@ public function testDefaults() : void
         $connection    = $this->buildConnection();
         $configuration = $this->buildConfiguration();
 
-        $container = $this->prophesize(ContainerInterface::class);
-        $container->has('config')->willReturn(false);
-        $container->has('doctrine.connection.orm_default')->willReturn(true);
-        $container->get('doctrine.connection.orm_default')->willReturn($connection);
-        $container->has('doctrine.configuration.orm_default')->willReturn(true);
-        $container->get('doctrine.configuration.orm_default')->willReturn($configuration);
+        $container = $this->getMockBuilder(ContainerInterface::class)->onlyMethods(['has', 'get'])->getMock();
```

Comment:
`$this->createMock()` suffices

Reply:
Addressed with bbe09841c12999e74a99b7af5d99804871fc90f2